sunpy/net/jsoc/jsoc.py

```diff
@@ -68,14 +68,14 @@ class JSOCClient(BaseClient):
 
     Notes
     -----
-    The full list of ``Series`` is available through this `site <http://jsoc.stanford.edu>`_.
+    The full list of ``Series`` is available through this `site <http://jsoc.stanford.edu>`__.
```

Comment:
Did you want to link direct to the page here?
```suggestion
    The full list of ``Series`` is available via ` this page <http://jsoc.stanford.edu/JsocSeries_DataProducts_map.html>`__.
```